From 6dde613c1a44731e017d262c2b5868dbe333da74 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Wed, 15 Mar 2017 09:00:19 -0400
Subject: [PATCH] Issue 49169 - Fix covscan errors(regression)

Description:  The change to vattr.c caused problems with the tests.
              Removing change.

https://pagure.io/389-ds-base/issue/49169

Reviewed by: one line commit rule

(cherry picked from commit 314e9ecf310d4ab8e8fc700bd5d3477d52e4fa19)
---
 ldap/servers/slapd/vattr.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ldap/servers/slapd/vattr.c b/ldap/servers/slapd/vattr.c
index 599b54e..ef4d7f2 100644
--- a/ldap/servers/slapd/vattr.c
+++ b/ldap/servers/slapd/vattr.c
@@ -754,9 +754,9 @@ slapi_vattr_values_get_sp(vattr_context *c,
   if (use_local_ctx) {
     /* slapi_pblock_destroy cleans up pb_vattr_context, as well */
 	slapi_pblock_destroy(local_pb);
-	ctx->pb = NULL;
+  } else {
+	vattr_context_ungrok(&c);
   }
-  vattr_context_ungrok(&ctx);
   return rc;
 }
 
-- 
2.9.3

